### PR TITLE
Added container's network links in events

### PR DESCRIPTION
**Pull request by cecton**

### CODE REVIEW

**Review by fr0gs — APPROVED** [review]
Code looks ok for me.

**Comment by earthquakesan** [comment]
Any changes to events2rdf.py should be provided with adding the corresponding elements to the events.owl ontology (it's in the docs). Code looks fine, however it is not aligned with the ontology at the moment and it breaks the changes I made previously. I kindly ask you to read events.owl. We can have a skype call if you need more insights why I designed it this way.

P.S. Providing triples with ontology is like exposing API without description/contract. The usage will be very cumbersome if not impossible (depends on complexity, which tends to grow with time). If the link between ontology and actual implementation will deteriorate over time, this logger will be useless for the community.

**Comment by cecton** [comment]
Ok sorry I'm not familiar with that process. I will do my best to add this doc.

**Comment by cecton** [comment]
@earthquakesan done. The doc doesn't really match the triple at the moment but I suppose we can change that later when the ontology is properly defined, right?

**Comment by earthquakesan** [comment]
Ontology is properly defined already. I did a mistake and assigned a property from network to container in the code. Other than that it is aligned with all the new changes done after migration.

File: docs/docker-engine-events/events.owl [hunk 1]
```diff
@@ -201,6 +201,13 @@ dockevent:networkIp a owl:DatatypeProperty ;
         rdfs:isDefinedBy  dockevent: ;
         rdfs:label        "network IP"@en .
 
+dockevent:networkLink a owl:DatatypeProperty ;
```
Comment (earthquakesan) [inline docs/docker-engine-events/events.owl:204]: DOCKCONTAINER_NETWORK.link is used in the code, but here is completely different prefix defined.

Comment (earthquakesan) [reply docs/docker-engine-events/events.owl:204]: event2rdf.py needs another revision (i.e. reverting back to the changes I did with the alignment and fixing that problem with networkIds, maybe I missed something else there?)

**Comment by cecton** [comment]
@earthquakesan ok fixed. Forgive me if I'm totally wrong, this is not really my cup of tea. So now I'm using in the code the predicate in the dockevent namespace. So I aligned the code to the ontology I described. Correct?

**Comment by earthquakesan** [comment]
yes, looks good now, thanks!

**Comment by fr0gs** [comment]
Ok, I will work with this for now then.

**Comment by cecton** [comment]
damn! I pushed a fixup XD

**Comment by cecton** [comment]
Nobody saw anything right? Everything is fine, I haven't pushed force the master branch, I'm innocent.

**Pull request merged**